src/resources/auth/controller.js

```diff
@@ -0,0 +1,41 @@
+const prisma = require('../..//utils/dbClient')
+const bycript = require('bcrypt')
+const createToken = require('../../utils/authentification');
+const crypto = require('crypto')
+
+const rounds = 9;
+
+async function signup(req, res){ 
+
+    const { email: userEmail, password : userPassword } = req.body
+    
+    const hashedPass = await bycript.hash(userPassword, rounds)
+
+    const random_uname = crypto.randomBytes(15).toString('hex');
```

Comment:
What's the purpose of the username? Is it meant to be a unique identifier?

Reply:
yes, that's what I ve thought